packages/web-components/src/components/footer/__stories__/README.stories.mdx

```diff
@@ -0,0 +1,178 @@
+    </dds-footer-nav-group>
+    <dds-footer-nav-group title-text="Information for...">
+      <dds-footer-nav-item href="https://developer.ibm.com/">Developers</dds-footer-nav-item>
+      <dds-footer-nav-item href="https://www.ibm.com/partnerworld/public">Business</dds-footer-nav-item>
+    </dds-footer-nav-group>
+  </dds-footer-nav>
+  <dds-locale-button></dds-locale-button>
+  <dds-legal-nav>
+    <dds-legal-nav-item href="https://www.ibm.com/contact/us/en/>Contact IBM</dds-legal-nav-item>
+    <dds-legal-nav-item href="https://www.ibm.com/privacy/us/en/">Privacy</dds-legal-nav-item>
+    <dds-legal-nav-item href="https://www.ibm.com/us-en/legal">Terms of use</dds-legal-nav-item>
+  </dds-legal-nav>
+</dds-footer>
+```
+
+> 💡 See [`<dds-locale-modal>` doc](/?path=/docs/components-locale-modal--default) for generating the content for locale modal.
+
+## `<dds-footer>`
+
+`<dds-footer>` works as the shell in footer.
+
+<Props of="dds-footer" />
+
+## `<dds-footer-logo>`
+
+`<dds-footer-logo>` shows the IBM brand logo in footer.
+
+<Props of="dds-masthead-logo" />
+
+## `<dds-footer-nav>`
+
+`<dds-footer-nav>` works as the primary collection of links in footer.
+
+<Props of="dds-footer-nav" />
+
+## `<dds-footer-nav-group>`
+
+`<dds-footer-nav-group>` works as a group of links in `<dds-footer-nav>`.
+
+<Props of="dds-footer-nav-group" />
+
+## `<dds-footer-nav-item>`
+
+`<dds-footer-nav-item>` works as a link in `<dds-footer-nav>`.
+
+<Props of="dds-footer-nav-item" />
+
+## `<dds-legal-nav>`
+
+`<dds-legal-nav>` works as the secondary collection of links in footer.
+
+<Props of="dds-legal-nav" />
+
+## `<dds-legal-nav-item>`
+
+`<dds-legal-nav-item>` works as a link in `<dds-legal-nav>`.
+
+<Props of="dds-legal-nav-item" />
+
+## `<dds-locale-button>`
+
+`<dds-locale-button>` works as a launcher of locale modal (`<dds-locale-modal>`).
+
+<Props of="dds-locale-button" />
+
+## Stable selectors
+
+See [our README](https://github.com/carbon-design-system/ibm-dotcom-library/tree/master/packages/web-components#stable-selectors-for-analytics-and-integratione2e-testing-in-web-components) to see how Web Components selector and `data-autoid` should be used.
+
+| Web Components selector  | Compatibility selector                 | Description |
+| ------------------------ | -------------------------------------- | ----------- |
+| `<dds-footer>`           | `data-autoid="dds--footer"`            | Component   |
+| `<dds-footer-nav>`       | `data-autoid="dds--footer-nav"`        | Component   |
+| `<dds-footer-nav-group>` | `data-autoid="dds--footer-nav-group"`  | Component   |
+| `<dds-footer-logo>`      | `data-autoid="dds--footer-logo"`       | Component   |
+| `<dds-locale-button>`    | `data-autoid="dds--footer-locale-btn"` | Interactive |
+| `<dds-legal-nav>`        | `data-autoid="dds--legal-nav"`         | Component   |
+
+<Description markdown={corsproxy} />
+
+## Contributing
```

Comment:
Don't need the extra `## Contributing` header, since the markdown include already comes with a heading.

Reply:
I see thanks - Updated.